scripts/validate-nimbus-file.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
```

Comment:
Are we still using `./scripts`? I thought this was deprecated in favor of `./src/scripts`

Reply:
`./src/scrips` sounds good to me — moved the script in [c153dca](https://github.com/mozilla/blurts-server/pull/4861/commits/c153dca5b67d535d3955bc69d55ab35104d4198c).